dashboard/app/models/sections/section.rb

```diff
@@ -193,7 +193,16 @@ def summarize
       linkToAssigned: link_to_assigned,
       numberOfStudents: students.length,
       linkToStudents: "#{base_url}#{id}/manage",
-      sectionCode: code
+      code: code,
+      stage_extras: stage_extras,
+      pairing_allowed: pairing_allowed,
+      login_type: login_type,
+      course_id: course_id,
+      script: {
+        id: script_id,
+        name: script.try(:name),
+      },
```

Comment:
Valid_assignments already has name by script_id. Ideally I think this would just return script_id, and components would then look up name by script. If that's untenable, returning both is probably ok here.